app/views/upload.scala.html

```diff
@@ -27,25 +27,38 @@ <h2 class="govuk-error-summary__title" id="error-summary-title">
         </div>
         <form id="file-upload-form" data-consignment-id="@consignmentId">
             <div class="govuk-form-group">
-                <label class="govuk-label" for="file-selection">
-                    @Messages("upload.selectFolder")
-                </label>
-                <input type="hidden" class="file-id-data" name="file-id-data" />
-                <input type="file" id="file-selection" name="files" class="govuk-file-upload" webkitdirectory>
-
-                <details class="govuk-details" data-module="govuk-details">
-                    <summary class="govuk-details__summary">
-                        <span class="govuk-details__summary-text">
-                            @Messages("upload.fileExtensionTitle")
-                        </span>
-                    </summary>
-                    <div class="govuk-details__text">
-                        @Messages("upload.fileExtensionSummary")
+                <div class="drag-and-drop">
+                    <div class="govuk-summary-list"></div>
+                    <div>
+                        <div class="govuk-form-group">
+                            <div class="drag-and-drop__dropzone">
+                            <input type="file" id="file-selection" name="files" class="govuk-file-upload drag-and-drop__input" webkitdirectory>
```

Comment:
This line should be indented by one more tab.

Reply:
Ah, yes; thanks!